hybrasyl/Objects/WorldObject.cs

```diff
@@ -53,10 +54,13 @@ public class WorldObject : IQuadStorable
         public World World { get; set; }
         public ushort DialogSprite { get; set; }
 
+        public ConcurrentDictionary<string, dynamic> EphemeralStore { get; set; }
+
         public WorldObject()
         {
             Name = string.Empty;
             ResetPursuits();
+            EphemeralStore = new ConcurrentDictionary<string, dynamic>();

```

Comment:
oh no, dynamics are the devil.

Reply:
Lua is a weakly / dynamically typed language, this is for mostly passing values between Lua and C#. `dynamic` is appropriate here.